test/utf8/OS_2977448.js

```diff
@@ -1,8 +1,14 @@
-//-------------------------------------------------------------------------------------------------------
-// Copyright (C) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
-//-------------------------------------------------------------------------------------------------------
-
-function TrimStackTracePath(line) {
-    return line && line.replace(/\(.+\\test.StackTrace./ig, "(");
-}
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+try {
```

Comment:
To be specific, the actual refactoring here deleted multiple copies of this script across full and core and moved one definitive copy to UnitTestFramework. Here's what the command line has to say:

```
>git show --name-status | grep --color StackTracePath
R083    test/Error/TrimStackTracePath.js        test/UnitTestFramework/TrimStackTracePath.js
D       test/inlining/TrimStackTracePath.js
R064    test/StackTrace/TrimStackTracePath.js   test/utf8/OS_2977448.js
```

---
In reply to: [107290021](https://github.com/Microsoft/ChakraCore/pull/2726#discussion_r107290021) [](ancestors = 107290021,107275264)